doc/transform_methods/onion_dasch.rst

```diff
@@ -0,0 +1,67 @@
+Onion Peeling (Dasch)
+=====================
+
+
+Introduction
+------------
+
+The "Dasch onion peeling" deconvolution algorithm is one of several
+described in the Dasch [1] paper. See also the ``two_point`` and 
+``three_point`` descriptions.
+
+How it works
+------------
+
+In the onion-peeling method the projection is approximated by rings
+of constant property between 
+:math:`r_j - \Delta r/2` and :math:`r_j + \Delta r/2` for each data 
+point :math:`r_j`.
+
+The projection data is given by :math:`P(r_i) = \Delta r \sum_{j=i}^\infty W_{ij} F(r_j)`
+
+where 
+
+.. math:: W_{ij} = 0 \, \, (j < i) 
+
+       \sqrt{(2j+1)^2 - 4i^2} \, \, (j=i)
+
+       \sqrt{(2j+1)^2 - 4i^2} - \sqrt{(2j-1)^2 - 4i^2} \, \, (j > i)
+
+
+The onion-peeling deconvolution function is: :math:`D_{ij} = (W^{-1})_{ij}`.
+
+
+When to use it
+--------------
+
+This method is simple and computationally very efficient. The article
+states that it has less smoothing that other methods (discussed in Dasch).
+
+
+How to use it
+-------------
+
+To complete the inverse transform of a full image with the ``onion_dasch`` method, simply use the :class:`abel.Transform` class: ::
+
+    abel.Transform(myImage, method='onion_dasch', direction='inverse').transform
+
+Note that the forward Three point transform is not yet implemented in PyAbel.
```

Comment:
@DanHickstein Not in the 1992 paper (or anywhere else to the best of my knowledge).